test/spec.js

```diff
@@ -376,6 +376,37 @@ describe('Trello', function () {
         });
     });
 
+    describe('addMemberToBoard', function() {
+        var query;
+        var data;
+        var post;
+
+        beforeEach(function (done) {
+            sinon.stub(restler, 'put', function (uri, options) {
+                return {once: function (event, callback) {
+                    callback(null, null);
+                }};
+            });
+
+            trello.addMemberToBoard('boardId', 'memberId', 'type', function () {
```

Comment:
I know you're following the convention of the tests in this file so far, but considering the valid values of type are `'normal','admin','observer'` this test case could be more realistic, because `type = 'type'` probably wouldn't work in real life.